nuitka/tree/ReformulationAssignmentStatements.py

```diff
@@ -1228,18 +1235,50 @@ def buildTypeVarNode(node, source_ref):
 def buildTypeAliasNode(provider, node, source_ref):
     """Python3.12 or higher, type alias statements."""
 
-    assert not node.type_params, node.type_params
-    type_alias_node = ExpressionTypeAlias(
-        type_params=buildNodeTuple(provider, node.type_params, source_ref),
-        value=buildNode(provider, node.value, source_ref),
-        source_ref=source_ref,
-    )
+    type_alias_name = mangleName(node.name.id, provider)
+
+    def createTypeExpression():
```

Comment:
What I don't like as much is that creates an outline function even if it becomes a simple return. Optimization will remove that, but it creates a bit of churn in verbose output I bet and produce a "change" which will though only make a difference if there is ever not more than one micro pass, but still. I probably want to make that outlike optional and just create the alias directly if there are no generic types.